list.go

```diff
@@ -21,15 +22,14 @@ const formatOptions = `table or json`
 // containerState represents the platform agnostic pieces relating to a
 // running container's status and state
 type containerState struct {
-	// ID is the container ID.
+	// ID is the container ID
 	ID string `json:"id"`
-
-	// InitProcessPid is the init process id in the parent namespace.
+	// InitProcessPid is the init process id in the parent namespace
 	InitProcessPid int `json:"pid"`
-
 	// Status is the current status of the container, running, paused, ...
 	Status string `json:"status"`
-
+	// Bundle is the path on the filesystem to the bundle
+	Bundle string `json:"bundle"`
```

Comment:
Bundle is correct here.  Note the use of the term bundle in the runc main useage.